benchmarks/testminer.py

```diff
@@ -638,10 +638,12 @@ def get_result_data(self, test_job_id):
         return (json_attachments[0][0], base64.b64decode(json_attachments[0][1]))
 
     def get_environment_name(self, metadata):
-        wanted = ('device', 'mode', 'core')
-        data = [ str(metadata[key]) for key in wanted if key in metadata ]
-        if len(data) == len(wanted):
-            return '-'.join(data)
+        wanted = ('device', 'mode', 'core', 'compiler-mode')
```

Comment:
well we could but all that code is already very ART-specific so it looks like a lost cause to me